users/faq.md

```diff
@@ -1 +1,33 @@
 # FAQ
+## Related to usage in Planetariums
+1. How do I get a fulldome (fisheye) output from OpenSpace?
+
+When you first start up OpenSpace, you get a screen where you can choose a Profile and choose Window Options. Window Options is where you can choose to use a fisheye configuration. 
+
+2. How can I save screenshots in a resolution higher than my screen resolution? For example, my computer screen can only display 1920x1080, but I want to capture 4096x4096 pngs using the session recording function.
+
+You can choose to display a different resolution, and render at a different resolution. The single.json Window options configuration file is an example which has a display resolution of 1280x720 (the "size" parameter) but renders pngs at 2560x1440 (the "res" parameter). These configuration files are located in the 'config' directory - you can write your own for your desired resolution.
+
+3. How can I get rid of the text and GUI elements in my planetarium display?
+
+There are several ways to do this. For example, you could
+(a) press TAB and SHIFT+TAB to toggle visibility of the GUI and text elements respectively, or
+(b) create or use a Window options configuration file which renders the GUI and the data visualization separately on different windows, for example the single_fisheye_gui.json configuration file or
+(c) create a [html control page](https://docs.openspaceproject.com/en/latest/how-to/html-controls-setup/index.html) and use that to control the navigation, hiding the GUI.
+
+4. How do I ensure that all the data sets I need are loaded before playing back a recorded path? Can I play back without an internet connection?
+
+Go into the openspace.cfg file and set ModuleConfigurations->GlobeBrowsing->MRFCacheEnabled to true. Run OpenSpace again it will cache the globebrowsing data for planet/moon locations you visit. Then later, it should use the cached data for those same locations. There is also an Offline profile that you can select from the launcher, and see if that has the content you want.
```

Comment:
```suggestion
Go into the `openspace.cfg` file and set {menuselection}`ModuleConfigurations --> GlobeBrowsing --> MRFCacheEnabled` to `true`. Run OpenSpace again it will cache the globe browsing data for planet/moon locations you visit. Then later, it should use the cached data for those same locations. There is also an `offline` profile that you can select from the launcher, and see if that has the content you want.
```